setup.py

```diff
@@ -3,7 +3,7 @@
 
 import pdfminer as package
 
-requires = ['six', 'pycryptodome']
+requires = ['six', 'pycryptodome', 'pygame']
```

Comment:
Again, what is `pygame` needed for?

Reply:
A function from `unitest` is called [here](https://github.com/mawoqiw/pdfminer.six/blob/4bdb3ba8cc79259336f797d00d9a48e93c53bdba/pdfminer/ccitt.py#L581) without an import.
`pygame` is not listed among requirements but is called [here](https://github.com/mawoqiw/pdfminer.six/blob/4bdb3ba8cc79259336f797d00d9a48e93c53bdba/pdfminer/ccitt.py#L585) and [here](https://github.com/mawoqiw/pdfminer.six/blob/4bdb3ba8cc79259336f797d00d9a48e93c53bdba/pdfminer/ccitt.py#L599)

So far I haven't found much space for improvement in `psparser.py,  other than this, either.

Reply:
@mawoqiw I think we should not add pygame as a dep... instead it should be removed. When was this introduced?

Reply:
This  `main()` should be removed entirely and moved to some testing script outside of the main library code  and we cannot add pygame as a runtime dep

Reply:
You are right @pombredanne. `pygame` is indeed not required to use Cython, thanks. 
With regards to `main()` function I would rather deal with it in a different merge request. Do you have any idea what is the purpose of that code?